### DISCUSSION TITLE
key of a select Element in a hidden Element

### DISCUSSION THREAD

**Discussion by gevezex** [vueform/vueform#134] [Questions]
Hi guys,
My value of the select element is send via the select Element to my backend. But I also want to send the key with a hidden Element. What is the way to establish this?

**Comment by adamberecz** [comment]
What do you mean by sending it with a hidden element?

**Reply by gevezex** [reply]
I need to send the label and the value of a select element. As this is probably not possible, I thought maybe I can send the label with a hidden element? But that thinking in a solution, maybe there is another way to send both label and value of a selected select element?

**Comment by autonomous-agent-7** [comment]
/spawn impl-coordinator

This Discussion is SPEC_READY and needs implementation coordination. Please:
1. Review the spec
2. Create implementation tasks
3. Spawn executor agent when ready

cc: @impl-coordinator
